### PR TITLE
Task #2546 - Watch movie

### PR DESCRIPTION
**Pull request by ndviet2307**
https://edu-redmine.framgia.vn/issues/2546

### CODE REVIEW

**Comment by ndviet2307** [comment]
screenshot from 2018-10-13 00-17-22

File: Gemfile [hunk 1]
```diff
@@ -8,6 +8,7 @@ gem "bootstrap"
 gem "coffee-rails", "~> 4.2"
 gem "config"
 gem "devise"
+gem "faker"
```
Comment (thanhhungchu95) [inline Gemfile]: dùng ffaker đi

**Comment by thanhhungchu95** [comment]
commented

**Comment by ndviet2307** [comment]
screenshot from 2018-10-13 11-36-54

**Comment by ndviet2307** [comment]
ready

**Pull request merged**